aslam_cv_calibration/src/target-aprilgrid.cc

```diff
@@ -36,7 +36,7 @@ TargetAprilGrid::TargetConfiguration::fromYaml(const std::string& yaml_file) {
     target_config.tag_inbetween_space_meter =
         relative_tag_spacing * target_config.tag_size_meter;
     CHECK_GT(target_config.tag_inbetween_space_meter, 0.0);
-  } catch (const std::exception& ex) {
+  } catch (const YAML::Exception& ex) {
```

Comment:
what happens to all other exceptions?

Reply:
@floriantschopp I guess there should not be any other exceptions than the one of not being able to read the yaml file?

Reply:
Agree. But in the case there is some, is it automatically thrown or do you have to do it manually by 
```cpp
catch(...) {
throw();
}
```
I'm not sure tbh.

Reply:
I guess the exception will be thrown by ``YAML::LoadFile` [here](https://github.com/jbeder/yaml-cpp/blob/master/src/parse.cpp#L35).

Reply:
@floriantschopp The program should terminate if an exception is thrown that you don't catch. This should be fine here as we don't expect any other exceptions to be thrown.

Reply:
ok makes sense, thank you!